tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js

```diff
@@ -0,0 +1,231 @@
+module.exports = {
+  commands: {
+    /**
+     * @enum {string}
+     * @readonly
+     */
+    FileAction: Object.freeze({
+      download: 'download',
+      delete: 'delete',
+      restore: 'restore',
+      share: 'share',
+      rename: 'rename',
+      deleteImmediately: 'deleteImmediately'
+    }),
+
+    /**
+     * Action button selector
+     *
+     * @param {string} action
+     * @returns {string}
+     */
+    getActionSelector: function (action) {
+      const actionsDropdownSelector = this.elements.itemActionsDropdown.selector
+      const actionSelector = this.elements[action + 'ButtonInFileRow'].selector
+
+      return `${actionsDropdownSelector}${actionSelector}`
+    },
+    /**
+     * Get Selector for File Actions expander
+     *
+     * @param {string} fileName
+     * @param {string} elementType
+     * @returns {string} file action button selector
+     */
+    getFileActionBtnSelector: function (fileName, elementType = 'file') {
```

Comment:
that belongs in the filesList page Object, this button is not part of the action menu

Reply:
Yes,,moved to filesList pageObject!